### PR TITLE
MV-292 Feature/mobile videoroom home page

### PR DESCRIPTION
**Pull request by alexmaar**
It adds a mobile home page with a settings preview.

### CODE REVIEW

**Review by bblaszkow06 — COMMENTED** [review]
Awaiting reviews from others

File: assets/css/app.css [hunk 1]
```diff
@@ -110,10 +110,10 @@ Don't display screensharing control on mobile browsers
 
 @media (max-width: 768px) {
   .home-page {
-    background-image: url("/images/backgroundLeft.png");
+    background-image: url("/images/backgroundLeft.png"), url("/images/backgroundRight.png");
     background-repeat: no-repeat;
-    background-size: contain;
-    background-position: left -90px top 200px;
+    background-size: 30%, 40%;
+    background-position: left -50px top 80px, right -70px bottom 15px;
```
[the remark this answers is not in the document]
Comment (alexmaar) [reply assets/css/app.css:116]: will fix in a separate PR

**Review by pkrucz00 — COMMENTED** [review]
There are still some things that should be implemented (possibly in some other PR):
 1. On mobile clicking the input element and showing the keyboard covers the other inputs.
  
Screenshot_2023-02-14-12-38-55-15_40deb401b9ffe8e1df2f1cc5ba480b12
Screenshot_2023-02-14-12-38-20-90_40deb401b9ffe8e1df2f1cc5ba480b12

 2. The mobile browser header is not included in the viewport metric. Some content of the site can be not well rendered. It's necessary to use a [dynamic view port](https://caniuse.com/viewport-unit-variants) or implement some other solution.

**Comment by alexmaar** [comment]
> There are still some things that should be implemented (possibly in some other PR):
> 
> 1. On mobile clicking the input element and showing the keyboard covers the other inputs.
> 
> Screenshot_2023-02-14-12-38-55-15_40deb401b9ffe8e1df2f1cc5ba480b12 Screenshot_2023-02-14-12-38-20-90_40deb401b9ffe8e1df2f1cc5ba480b12
> 
> 2. The mobile browser header is not included in the viewport metric. Some content of the site can be not well rendered. It's necessary to use a [dynamic view port](https://caniuse.com/viewport-unit-variants) or implement some other solution.

I've already changed the viewport to dynamic. For the issues related to the mobile keyboard, I will create a separate task.

**Pull request merged**